data/faq.json

```diff
@@ -0,0 +1,110 @@
+        "@type": "Answer",
+        "text": "<p>A: We expect to see the community to work towards better integrations with cloud native technologies such as Kubernetes and Docker. Some of these integrations need to happen at the Java virtual machine (JVM) level. We expect the Jakarta EE community to work closely with the OpenJDK and Eclipse OpenJ9 team to provide support at the framework level as these JVM enhancements are made available.</p>"
+      }
+    },{
+      "@type": "Question",
+      "name": "What are the key intersections making sure the language and the EE platform are in lock-step?",
+      "acceptedAnswer": {
+        "@type": "Answer",
+        "text": "<p>A: The Jakarta EE release cadence and schedule have not yet been determined. Once the community decides what the correct cadence is, we can start discussions to ensure alignment with other communities like OpenJDK.</p>"
+      }
+    },{
+      "@type": "Question",
+      "name": "Who and how can one use Jakarta EE Compatibility Logo?",
+      "acceptedAnswer": {
+        "@type": "Answer",
+        "text": "<p>A: The first thing to understand is that the logo is not available for any specifications other than the Profiles (Full, Web). So, for example, implementing only Servlet does not permit the use of the Jakarta EE Compatibility logo by any organization or project.</p><p>If an open source implementation of the Jakarta EE 8 Web or Full Profile follows the TCK process and is in compliance with the Eclipse Foundation TCK license, the open source project can certainly declare that it is a Compatible Implementation.</p><p>In order to use the Jakarta EE Compatibility logo, an organization sponsoring the open source project must meet the membership eligibility requirements specified in the Jakarta EE Trademark Guidelines.</p><p>At a high level, the use of the Compatibility logo is limited to only members of the Jakarta EE Working Group who are licensees under the Jakarta EE Trademark License Agreement or approved guest members of the Working Group (who are also licensees under the Eclipse Foundation Trademark License).</p><p>We are working with our friends at the Apache Software Foundation to allow their projects to use the Compatibility logo at no charge. We are open to working with other open source foundations that are interested in making the logo available to their eligible projects.</p><p>We require membership in the Working Group because the Jakarta EE Compatibility logo and brand have value to our community and the industry. We rely on membership funding to financially support the management of the Jakarta EE specification process, compatibility rules, and developer outreach activities to drive awareness and adoption. Also, Working Group membership fees fund the promotion of the Jakarta EE Compatible brand and its value in the marketplace.</p><p>Refer to <a href='https://jakarta.ee/legal/trademark_guidelines/'>Jakarta EE Trademark Guidelines</a> for more details.</p>"
```

Comment:
I think we should remove the A: from the text. I am concern that this will be included in the search results.

Reply:
Emmm.. I'm actually hoping to keep the "A:" in https://jakarta.ee/about/faq/ , is it okay we remove them? If I add "A:" in the shortcode to all the answers, it will be weird to arrange the <p> and keep them in the same lines ... I'm still trying to find the best way to do so

Reply:
i was thinking that we should remove the A: completely...

Reply:
Actually if the community agrees, we do not need "Q" and "A" in front of every question and answer 🤔

Reply:
+1 Let's make the change... I am pretty sure I was the one that added those A: in the first place..

Reply:
@TanjaObradovic Any objections with us removing the A: before each answer on the FAQ page? I think it would be best to remove it for the SEO work that we are doing!

Reply:
removed.